extensions/kudo/client/src/main/java/io/fabric8/kudo/client/DefaultKudoClient.java

```diff
@@ -0,0 +1,53 @@
+package io.fabric8.kudo.client;
+
+import io.fabric8.kubernetes.client.BaseClient;
+import io.fabric8.kubernetes.client.Config;
+import io.fabric8.kubernetes.client.KubernetesClientException;
+import io.fabric8.kubernetes.client.dsl.MixedOperation;
+import io.fabric8.kubernetes.client.dsl.Resource;
+import io.fabric8.kudo.api.model.v1beta1.*;
```

Comment:
Let's just get rid of the `api.model` which feels redundant. 
We've applied this practice in the rest of the extensions and it feels so much better.

Reply:
modified and commit the code.